packages/flutter_hooks/lib/src/framework.dart

```diff
@@ -225,9 +226,26 @@ abstract class HookState<R, T extends Hook<R>> with Diagnosticable {
   @protected
   void initHook() {}
 
+  /// Register a callback for when [HookWidget] build is finished
+  /// If dispose method has been override, super.dispose or [removeDidBuildListener] must be called
+  void setDidBuildListener(VoidCallback callback) {
```

Comment:
This should likely be a getter/setter instead

Such that we do:

```dart
onDidBuild = () {

};

onDidBuild = null;
```

Reply:
This is done